File: api/src/main/java/moe/caa/multilogin/api/logging/impl/NullLogger.java
package moe.caa.multilogin.api.logging.impl;

import moe.caa.multilogin.api.logging.IMultiloginLogger;
import moe.caa.multilogin.api.logging.LogLevel;
import org.jetbrains.annotations.NotNull;
import org.jetbrains.annotations.Nullable;

public class NullLogger implements IMultiloginLogger {
    @Override
    public void log(@NotNull LogLevel level, @Nullable String message, @Nullable Throwable throwable) {
        // Do nothing.
    }
}

File: api/src/main/kotlin/moe/caa/multilogin/api/logger/bridge/NullLogger.kt
package moe.caa.multilogin.api.logger.bridge

import moe.caa.multilogin.api.logger.Level
import moe.caa.multilogin.api.logger.Logger
import org.jetbrains.annotations.ApiStatus

/**
 * 空日志
 */
@ApiStatus.Internal
object NullLogger : Logger {
    override fun log(level: Level, message: String?, throwable: Throwable?) {}
}